src/gui/transferlistfilterswidget.h

```diff
@@ -116,10 +111,10 @@ private slots:
 private:
     // These 4 methods are virtual slots in the base class.
     // No need to redeclare them here as slots.
-    virtual void showMenu(QPoint);
-    virtual void applyFilter(int row);
-    virtual void handleNewTorrent(BitTorrent::TorrentHandle *const torrent);
-    virtual void torrentAboutToBeDeleted(BitTorrent::TorrentHandle *const torrent);
+    virtual void showMenu(QPoint) override;
```

Comment:
can the `virtual` be striped off?

Reply:
Yeah, i'll fix these and the other comments.